mods/ra/rules/world.yaml

```diff
@@ -91,11 +91,11 @@ World:
 		FogVariants: shroud
 		Index: 255, 16, 32, 48, 64, 80, 96, 112, 128, 144, 160, 176, 192, 208, 224, 240, 20, 40, 56, 65, 97, 130, 148, 194, 24, 33, 66, 132, 28, 41, 67, 134, 1, 2, 4, 8, 3, 6, 12, 9, 7, 14, 13, 11, 5, 10, 15, 255
 		UseExtendedIndex: true
-	Country@0:
+	Country@0: # Mission country
```

Comment:
I'd say these comments are unnecessary. We only use `Selectable: False` for factions that are only used in missions and "special" maps as far as I know, so it should be obvious.
Also the utility will probably want to remove them anyway, which makes for even more changes on each run.